objects/src/notes/assets.rs

```diff
@@ -25,7 +26,7 @@ impl NoteAssets {
     // CONSTANTS
     // --------------------------------------------------------------------------------------------
     /// The maximum number of assets which can be carried by a single note.
-    pub const MAX_NUM_ASSETS: usize = 255;
+    pub const MAX_NUM_ASSETS: usize = MAX_ASSETS_PER_NOTE;
```

Comment:
Do we need to re-declare a new constant here? 

Could we use `MAX_ASSETS_PER_NOTE` directly?

Reply:
We could here, but this is more of a convenience for users of `NoteAssets` as they would be able to say `NoteAssetss::MAX_ASSETS_PER_NOTE` instead of importing an additional constant.